src/content/tools/pub/cmd/pub-token.md

```diff
@@ -42,33 +44,45 @@ Enter secret token: <Type token on stdin>
 ```
 
 :::note
-The token is input on stdin rather than as a command line option to avoid it
-ending up in the shell history such as `~/.bash_history`.
+The token takes input on `stdin` rather than as a command line option
+to keep the token out of the shell history.
 :::
 
-In a scripting situation you can store the secret in an environment variable and
-use `dart pub token add <hosted-url> --env-var <ENV_VAR_NAME>`.
+### Add a credential for all sessions
 
-```console
-$ dart pub token add https://other-package-repo.com/ --env-var TOKEN_VAR
-Requests to "https://other-package-repo.com/" will now be authenticated using the secret token stored in the environment variable "TOKEN_VAR".
-```
+In a script, you can store the secret in an environment variable.
+
+If you choose to store your secret in an environment variable,
+find a way to hide the secret from your shell history.
+To explore one way of doing this, consult [this post on Medium][zsh-post].
```

Comment:
```suggestion
If you choose to store your token in an environment variable,
[find a way][zsh-post] to hide the token from your shell history.
```

I think it's odd to call out a medium post that wasn't written by us. Suggestion deemphasizes it.